client/static_src/components/clients/client.vue

```diff
@@ -4,7 +4,7 @@
   :delete_perm="$perms.delete_client"
   :change_perm="$perms.change_client"
   :add_perm="$perms.add_client"
-  @modal="$emit('modal')"
+  @modal="showEntries"
```

Comment:
This is a quick way to access all time entries of a project. I think this a more common use case than editing the project.